windows/webview.cc

```diff
@@ -373,21 +373,24 @@ void Webview::RegisterEventHandlers() {
       &event_registrations_.contains_fullscreen_element_changed_token_);
 }
 
-void Webview::SetSurfaceSize(size_t width, size_t height) {
+void Webview::SetSurfaceSize(size_t width, size_t height, float scale_factor) {
   if (!IsValid()) {
     return;
   }
 
   if (surface_ && width > 0 && height > 0) {
-    surface_->put_Size({(float)width, (float)height});
+    scale_factor_ = scale_factor;
+    auto sw = (float)width * scale_factor;
+    auto sh = (float)height * scale_factor;
+    surface_->put_Size({sw, sh});
 
     RECT bounds;
     bounds.left = 0;
     bounds.top = 0;
-    bounds.right = static_cast<LONG>(width);
-    bounds.bottom = static_cast<LONG>(height);
+    bounds.right = static_cast<LONG>(sw);
+    bounds.bottom = static_cast<LONG>(sh);
 
-    if (webview_controller_->put_Bounds(bounds) != S_OK) {
+    if (webview_controller_->SetBoundsAndZoomFactor(bounds, scale_factor) != S_OK) {

```

Comment:
I tried on some way with `put_rasterizationscale` and didn't get any high resolution video image output.

Reply:
Changing the rasterization scale works fine for me. It could have been an issue with your Webview version.

Before (with `devicePixelRatio`=1 on a 27" 4K screen) :
![image](https://user-images.githubusercontent.com/720469/204085773-04b5e98f-75ae-445f-a862-66f089c51609.png)

After (with an effective `devicePixelRatio` of 1.75 on the same screen):
![image](https://user-images.githubusercontent.com/720469/204085881-7d6f26be-3ecf-4f2f-860f-257829ca5dac.png)

Reply:
@jnschulze That's weird, I still getting a blurry image with Webview version(107.0.1418.56) on a 27" 4K screen. I have no idea what is going on here. 😅

<img width="951" alt="2022-11-26 201402" src="https://user-images.githubusercontent.com/3091033/204088430-3aac6d96-9156-485a-bb15-1fbe9944e43c.png">

Reply:
Does manually setting an arbitrary `scaleFactor` (I added that property in this PR) have any effect?

Reply:
`devicePixelRatio = 1.0`
<img width="951" alt="1 0" src="https://user-images.githubusercontent.com/3091033/204091455-b0dc9315-c9f2-4c31-aa66-32b2354a45ae.png">

`devicePixelRatio = 1.5` （This is the default value corresponding to my Windows 150% scaling）
<img width="951" alt="1 5" src="https://user-images.githubusercontent.com/3091033/204091457-5ef9a6d1-116e-42bc-8127-7cf2bfc0886d.png">

`devicePixelRatio = 2.0`
<img width="951" alt="2 0" src="https://user-images.githubusercontent.com/3091033/204091456-894650e7-459e-4516-b9a0-904fd995d5b5.png">